scripts/Actions/Explore.ts

```diff
@@ -42,10 +42,21 @@ export function hideAddedAnime() {
 }
 
 // Hides anime that are not in your list.
-export function calendarShowAddedAnimeOnly() {
+export function calendarShowAddedAnimeOnly(arn: AnimeNotifier, element: HTMLInputElement) {
 	for(let anime of findAll("calendar-entry")) {
 		if(anime.dataset.added === "false") {
 			anime.classList.toggle("hidden")
 		}
 	}
+
+	const showUserList = !Array.from(document.getElementsByClassName("calendar-entry"))
```

Comment:
This should have used the result of `findAll` in line 46 instead of instructing the browser to find those elements again.